Cron fonctionne
Cron fonctionne
Cron avance
Cron avance
Cron roule
Traceback (most recent call last):
  File "/home/gabintel/g.prost/SDD/velou-/update.py", line 1, in <module>
    import update_dynamic_data_velou_bd
  File "/home/gabintel/g.prost/SDD/velou-/update_dynamic_data_velou_bd.py", line 57, in <module>
    update_dynamic_data_velou_bd()
  File "/home/gabintel/g.prost/SDD/velou-/update_dynamic_data_velou_bd.py", line 7, in update_dynamic_data_velou_bd
    with open('data_dynamique.json', 'r') as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'data_dynamique.json'
/home/gabintel/anaconda3/envs/velou/bin/python3
Données enregistrées dans le fichier 'data_dynamique.json'.
Données dynamiques récupérées avec succès.
Données enregistrées dans le fichier 'data_dynamique.json'.
Données dynamiques récupérées avec succès.
Données enregistrées dans le fichier 'data_dynamique.json'.
Données dynamiques récupérées avec succès.
